### PR TITLE
Fixed new message screen

### PR DESCRIPTION
**Pull request by Neatwhiskey**
Screenshot 2021-10-01 at 20 23 18

https://user-images.githubusercontent.com/44437144/135703271-7d334b78-b0e4-4257-96ca-39856837c02a.mp4

### CODE REVIEW

File: lib/ui/view/dm_user/dm_user_view.dart [hunk 1]
```diff
@@ -1,355 +1,335 @@
...
-                            fontWeight: FontWeight.w400)),
-                  ],
-                ),
-                const Text(
-                  'View Details',
-                  style:
-                      TextStyle(color: AppColors.greyishColor, fontSize: 12.0),
+        viewModelBuilder: () => DmUserViewModel(),
+        builder: (context, model, child) {
+          debugPrint(model.hasClickedMessageField.toString());
```
Comment (pauleke65) [inline lib/ui/view/dm_user/dm_user_view.dart:23]: remove the debug Print

Comment (Neatwhiskey) [reply lib/ui/view/dm_user/dm_user_view.dart:23]: Okay

File: lib/ui/view/dm_user/dm_user_view.dart [hunk 2]
```diff
@@ -1,355 +1,335 @@
...
+              ],
             ),
-          ),
+          );
+        });
+  }
 
-          // Stack(
-          //   children: [
+  Widget _onlineIndicator(int color) {
```
Comment (pauleke65) [inline lib/ui/view/dm_user/dm_user_view.dart:282]: please create a widgets directory and create a file and put this in there

File: lib/ui/view/dm_user/dm_user_view.dart [hunk 3]
```diff
@@ -1,355 +1,335 @@
...
-          //                             expands: true,
-          //                             maxLines: null,
-          //                             textAlignVertical:
-          //                                 TextAlignVertical.center,
-          //                             decoration: InputDecoration(
-          //                               hintText:
-          //                                   ,
-          //                               // suffixIcon: CustomStatus(isActive: true,
-          //                               //   data: '5'),
+  Widget _groupSeparator(String value) {
```
Comment (pauleke65) [inline lib/ui/view/dm_user/dm_user_view.dart:290]: also create a file in the widgets directory for this too

File: lib/ui/view/dm_user/dm_user_view.dart [hunk 4]
```diff
@@ -1,355 +1,335 @@
...
-          //                 )),
-          //           ],
-          //         ),
-          //       ),
-          //     )
-          //   ],
-          // ),
-        );
-      },
+  Widget _startMessage(String username) {
```
Comment (pauleke65) [inline lib/ui/view/dm_user/dm_user_view.dart:316]: Same for this too, widgets folder

File: lib/ui/view/dm_user/dm_user_view.dart [hunk 5]
```diff
@@ -1,355 +1,335 @@
...
+                                  itemOne.id.compareTo(itemTwo.id),
+                            ),
+                            const SizedBox(height: 40)
+                          ],
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+                Align(
```
Comment (pauleke65) [inline lib/ui/view/dm_user/dm_user_view.dart:144]: Is this the reply chat box from channels or threads?
Please create a file for it in the widgets directory

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Please make requested changes

**Comment by pauleke65** [comment]
Please upload video demo

**Pull request closed without merging**